src/helics/core/BrokerBase.hpp

```diff
@@ -85,12 +86,12 @@ class BrokerBase {
     bool observer{false};
 
   private:
-    std::atomic<bool> mainLoopIsRunning{
-        false};  //!< flag indicating that the main processing loop is running
+    /// flag indicating that the main processing loop is running
+    std::atomic<bool> mainLoopIsRunning{false};
     bool dumplog{false};  //!< flag indicating the broker should capture a dump log
     std::atomic<bool> forceLoggingFlush{false};  //!< force the log to flush after every message
-    bool queueDisabled{
-        false};  //!< flag indicating that the message queue should not be used and all functions
+    /// flag indicating that the message queue should not be used and all functions
+    bool queueDisabled{false};
     //!< called directly instead of distinct thread
```

Comment:
`called directly instead of distinct thread` looks like it should be moved up to join the other comment line above the variable declaration.

Reply:
I think I cleared this up in the latest commit